jenkins/L0_Test.groovy

```diff
@@ -755,17 +974,68 @@ def getMakoOpts(getMakoScript, makoArgs=[]) {
         }
     }
 
-    // Finally, convert the query to a json string
     def makoOptsJson = JsonOutput.toJson(makoOpts)
 
     // Print and return the Test DB Query as a JSON string
     echo "Test DB Mako opts: ${makoOptsJson}"
+    return makoOptsJson
+}
+
+def getMakoOpts(getMakoScript, makoArgs=[]) {
+    // We want to save a map for the Mako opts
+    def turtleOutput = ""
```

Comment:
Whats is turtle used for here?

We should get rid of it, and use pytest as much as possible to better local reproducing.